### DISCUSSION TITLE
Does snntorch need to encode the input data into a spike train?

### DISCUSSION THREAD

**Discussion by EliteSasuke** [jeshraghian/snntorch#120] [General]
* snntorch version:0.4.10
* Python version:3.7
* Operating System:Win 11

### Description
Hi Jason,
I didn't find the program code to encode the input data in tutorial 5, nor in snn.Leaky. Does this mean that the user can input data into the model in Tutorial 5 without coding the input data into spike train?

**Comment by jeshraghian** [comment]
Exactly. This has the same effect as applying a constant current injection (see tutorial 2 for plots on this).

Converted to a discussion rather than as an issue.
